pyproject.toml

```diff
@@ -53,21 +54,21 @@ force_grid_wrap = 2
 
 
 [tool.pylint.master]
-fail-under = 7.5
+fail-under = 6.2
```

Comment:
I think this one can be increased. I recommend setting it to the currently determined score, so deterioration can be avoided.